test/pipeline_test_runner.cc

```diff
@@ -391,15 +391,13 @@ TEST_CASE("PerPhaseTest") { // NOLINT
         filesSpan = filesSpan.subspan(numPackageFiles);
 
         trees = index(*gs, inputPackageFiles, handler, test);
-
-        // First run: only the __package.rb files. This populates the packageDB
-        package(*gs, workers, absl::Span<ast::ParsedFile>(trees), handler, assertions);
-        name(*gs, absl::Span<ast::ParsedFile>(trees), *workers);
     }
 
     auto nonPackageTrees = index(*gs, filesSpan, handler, test);
-    package(*gs, workers, absl::Span<ast::ParsedFile>(nonPackageTrees), handler, assertions);
+    name(*gs, absl::Span<ast::ParsedFile>(trees), *workers);
+    package(*gs, workers, absl::Span<ast::ParsedFile>(trees), handler, assertions);
     name(*gs, absl::Span<ast::ParsedFile>(nonPackageTrees), *workers);
+    package(*gs, workers, absl::Span<ast::ParsedFile>(nonPackageTrees), handler, assertions);
```

Comment:
Is it worth using the `buildPackageDB` and `validatePackagedFiles` specializations here?

Reply:
These helpers are pipeline_test_runner helpers and there is no alternative with different names.